Improve clarity (#22236)
mdn/content: files/en-us/web/javascript/guide/working_with_objects/index.md

### Using this for object references

> JavaScript has a special keyword, [`this`](/en-US/docs/Web/JavaScript/Reference/Operators/this), that you can use within a method to refer to the current object. For example, suppose you have 2 objects, `Manager` and `Intern`. Each object have their own `name`, `age` and `job`. In the function `sayHi()`, notice there is `this.name`. When added to the 2 objects, they can be called and print the message with their respective name.

Reviewer: This sentence looks a bit awkward to me overall (funny I didn't catch this in my recent rewrite). What about:

```suggestion
JavaScript has a special keyword, [`this`](/en-US/docs/Web/JavaScript/Reference/Operators/this), that you can use within a method to refer to the current object. For example, suppose you have 2 objects, `Manager` and `Intern`. Each object has its own `name`, `age` and `job`. In the function `sayHi()`, notice the use of `this.name`. When added to the 2 objects, the same function will print the message with the name of the respective object it's attached to.
```

Author: I think this suggested change version is clearer for a non-native speaker.
